egs/gigaspeech/s5/run.sh

```diff
@@ -1,143 +1,134 @@
 #!/usr/bin/env bash
-# Copyright 2021 Xiaomi Corporation (Author: Yongqing Wang)
+# Copyright 2021  Xiaomi Corporation (Author: Yongqing Wang)
+#                 Seasalt AI, Inc (Author: Guoguo Chen)
 # Apache 2.0
 
-. ./cmd.sh
-. ./path.sh
+set -e
+set -o pipefail
+
+. ./cmd.sh || exit 1;
+. ./path.sh || exit 1;
+. ./utils/parse_options.sh || exit 1;
```

Comment:
It seems that `. ./utils/parse_options.sh` have to be after definitions of all those args and options, or it will return `run.sh: invalid option --stage`

Reply:
you are right, my bad, fixed.